### PR TITLE
fix: explicitly expect one message in timelock execute_execute_proposal #ntrn-129

### PR DESCRIPTION
**Pull request by NeverHappened**
https://hadronlabs.atlassian.net/browse/NTRN-129

### CODE REVIEW

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6641224559 (check!)

File: contracts/subdaos/cwd-subdao-timelock-single/src/contract.rs [hunk 1]
```diff
@@ -176,17 +176,25 @@ pub fn execute_execute_proposal(
             .querier
             .query_wasm_smart(proposal_module, &ProposalQueryMsg::Config {})?;
 
+        // expect exactly one `ExecuteMsg::ExecuteTimelockedMsgs` message
+        if proposal.msgs.len() != 1 {
```
Comment (pr0n00gler) [inline contracts/subdaos/cwd-subdao-timelock-single/src/contract.rs:180]: Let's explicitly add a check somewhere here that the message is really `ExecuteMsg::ExecuteTimelockedMsgs` and nothing else

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6656331425 (check!)

**Review by pr0n00gler — APPROVED** [review]

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6720271240

**Review by sotnikov-s — APPROVED** [review]

**Comment by NeverHappened** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6734438598

**Pull request merged**